packages/apps/docs/next.config.mjs

```diff
@@ -20,7 +20,7 @@ import redirectsConfig from './redirects.mjs';
 const options = {
   // Use one of Shiki's packaged themes
   theme: {
-    light: 'github-light',
+    light: 'github-dark',
```

Comment:
all dark? not a mistake?

Reply:
The dark and light modes intentionally have the same code (blue) styling. This is to simplify the code block stylings and create a clear distinction between content and code.